robocop/checkers/misc.py

```diff
@@ -339,6 +348,27 @@
             END
         """,
     ),
+    "0919": Rule(
+        rule_id="0919",
+        name="unused-argument",
+        msg="Keyword '{{ arg }}' argument is not used",
```

Comment:
I think I would change it to:
```suggestion
        msg="Keyword argument '{{ arg }}' is not used",
```
because the previous one makes me think that it is about an argument from keyword named {{ arg }}, not an argument names  {{ arg }}.